corehq/motech/repeaters/models.py

```diff
@@ -196,17 +196,14 @@ def all_ready(self):
                 .filter(repeat_records_ready_to_send))
 
 
-class RepeaterStub(models.Model):
-    """
-    This model links the SQLRepeatRecords of a Repeater.
-    """
+class SQLRepeater(models.Model):
```

Comment:
The typical pattern is to specify a name without the `sql_` prefix in `Meta.db_table`. Would that be possible here?

See the `db_table` notes in https://commcare-hq.readthedocs.io/couch_to_sql_models.html#pr-1-add-sql-model-and-migration-management-command-write-to-sql